packages/iTwinUI-react/src/core/Dialog/DialogContext.tsx

```diff
@@ -39,6 +39,11 @@ export type DialogContextProps = {
    * @default false
    */
   preventDocumentScroll?: boolean;
+  /**
+   * Flag whether dialog is draggable.
```

Comment:
Maybe add additional info, that you need `DialogWrapper` or put even an example in here?

Reply:
But you don't `DialogWrapper` to make dialog draggable. Or you mean to mention `DialogWrapper` here?

Reply:
Yeah, sorry, meant leaving info about availability of `DialogWrapper` if you need to have dialog draggable relative to container

Reply:
Mentioned `DialogWrapper` in comments.